pennylane/ops/qubit/arbitrary_ops.py

```diff
@@ -481,3 +481,129 @@ def _decomposition_with_one_worker(control_wires, target_wire, work_wire):
         ]
 
         return gates
+
+
+class Hermitian(Observable):
```

Comment:
I think this one fits well here though (even if it's an observable). If the file is renamed to e.g. `general_ops` or `unitary_ops` or `matrix_ops`, or something better.

Reply:
Think I like `matrix_ops`.